app/controllers/partners/dashboards_controller.rb

```diff
@@ -4,6 +4,8 @@ class DashboardsController < BaseController
 
     protect_from_forgery with: :exception
 
+    before_action :require_partner
```

Comment:
Shouldn't this be in the partner base controller? I feel like every page under there needs a current partner, no?

Reply:
oh, I didn't realize BaseController is Partners::BaseController. I'll move it!